DOC-2363: Added new `A11ycheckStart`, `A11ycheckStop`, `A11ycheckIgnore`, `A11ycheckRepair` and `A11ycheckShowDetails` events.. (#3245)
tinymce/tinymce-docs: modules/ROOT/pages/7.1-release-notes.adoc

The following premium plugin updates were released alongside {productname} 7.1.

=== Accessibility Checker

> The {productname} 7.1 release includes an accompanying release of the **Accessibility Checker** premium plugin.

Reviewer: ```suggestion
The {productname} {release-version} release includes an accompanying release of the **Accessibility Checker** premium plugin.
```

**Accessibility Checker** includes the following improvement.

==== Added new `A11ycheckStart`, `A11ycheckStop`, `A11ycheckIgnore`, `A11ycheckRepair` and `A11ycheckShowDetails` events.
// #TINY-10777

> {productname} introduces five new events within the **Accessibility Checker** premium plugin. These events capture usage statistics, offering valuable insights into the tool's internal usage for customers.

Reviewer: ```suggestion
{productname} {release-version} introduces five new events within the **Accessibility Checker** premium plugin. These events capture usage statistics, offering valuable insights into the tool's internal usage for customers.
```

The events include:

[cols="1,2",options="header"]
|===
|Name |Description
|A11ycheckStart |Fired when the Accessibility Checker is `opened`
|A11ycheckStop |Fired when the Accessibility Checker is `closed`

> |A11ycheckIgnore |Fired when the `Ignore` button is toggled
> |A11ycheckRepair |Fired when the `Repair` button is toggled
> |A11ycheckShowDetails |Fired when the `Click for more info` button is toggled

Reviewer: ```suggestion
|A11ycheckIgnore |Fired when the `Ignore` button is pressed
|A11ycheckRepair |Fired when the `Repair` button is pressed
|A11ycheckShowDetails |Fired when the `Click for more info` button is pressed
```

|===

> [NOTE]

Reviewer: doesn't the integrator choose how to handle the event being fired?

Author: Good point @FarzadHayat 
@spocke - Should we consider adding details on adding callback function to there integration if they want to initialise these events?

Reviewer 2: Yes how to listen to events should be a general thing we document and what they do in the handle could be anything.

> [NOTE]
> These events will only be visible within the developer console when each `event` is triggered.
> 
> For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].

Reviewer: ```suggestion
For information on the **Accessibility Checker** plugin, see: xref:a11ychecker.adoc[Accessibility Checker].

For information on handling events in {productname}, see: xref:events.adoc[Events].
```
